src/Database/PostUpdate.php

```diff
@@ -5,11 +5,9 @@
 namespace Friendica\Database;
 
 use Friendica\Core\Config;
-use Friendica\Database\DBM;
 use Friendica\Model\Contact;
 use Friendica\Model\Item;
 use Friendica\Model\PermissionSet;
-use dba;
```

Comment:
Because `Friendica\Database\dba` is the same namespace as `Friendica\Database\PostUpdate`.

I removed a bunch of other extraneous `use` statements in `Friendica\Model` classes as well.